mcu-util/src/main.rs

```diff
@@ -170,16 +169,16 @@ async fn execute(args: Args) -> Result<()> {
                     })?;
                 }
             }
-            Ok(())
         }
         SubCommand::SecureElement(opts) => match opts {
             SecureElement::PowerCycle => {
                 orb.borrow_mut_sec_board()
                     .power_cycle_secure_element()
-                    .await
+                    .await?
             }
         },
     }
+    orb_tasks.join().await
```

Comment:
this doesn't seem to return.
we would need to explicitly kill the can_rx task I think 🤔 

try with `orb-mcu-util info`, it never returns

Reply:
fixed